frontend/src/util/validators.js

```diff
@@ -241,6 +262,17 @@ function validateAlpha(input, length) {
   return !/[^a-zA-Z]/.test(input)
 }
 
+/**
+ * Validate that input is allowed length and that it contains alphabets and spaces
+ */
+function validateAlphaWithSpaces(input, length) {
```

Comment:
What is the purpose of this method? Also, Alphanumeric validator seems not working for city and province. Please have a look.